CMakeLists.txt

```diff
@@ -69,10 +69,9 @@ SET(CMAKE_CXX_FLAGS "-Wall -Wno-sign-compare -g -trigraphs -Wno-unused-function
 ### Build Tests OR lv2
 ######################
 IF( FABLA2_TESTS )
-  
+
   ADD_DEFINITIONS( "-DFABLA2_COMPONENT_TEST" )
-  
-  SET(CMAKE_CXX_FLAGS "-g -trigraphs -Wl,-z,nodelete  -Wl,--no-undefined") # -fsanitize=address
```

Comment:
Not really sure why this was here, these flags were all set on line 61 already.
Or should some CXX flags not be set when the tests are enabled?